content/doc/book/managing/index.adoc

```diff
@@ -8,7 +8,7 @@ layout: chapter
 :sectanchors:
 :toc: left
 
-= Manage Jenkins
+= Managing Jenkins
```

Comment:
The only reason I like this title as "Manage Jenkins" is because that's the exact text used inside the app